getting-started/templates/AWS/aws_supplemental_values.yml

```diff
@@ -58,6 +90,79 @@ dataframeservice:
   ingress:
```

Comment:
This will likely conflict with my change to update the ingress configuration:

https://github.com/ni/install-systemlink-enterprise/pull/273

We'll need to be careful not to break the config on commit.

Reply:
I'm going to go ahead and merge this one.